turbinia/processors/mount_local.py

```diff
@@ -51,15 +51,35 @@ def PreprocessMountDisk(evidence):
 
   evidence.mount_path = tempfile.mkdtemp(prefix='turbinia', dir=mount_prefix)
 
-  if hasattr(evidence, 'mount_partition') and evidence.mount_partition:
-    src_path = '{0:s}-part{1:d}'.format(
-        evidence.local_path, evidence.mount_partition)
-  else:
-    src_path = evidence.local_path
+  # Two steps: first use losetup to parse the RawDisk eventual partition table
```

Comment:
Why eventual?  do you mean potential?

Reply:
it may or may not be present. Stupid false-friends =)